### PR TITLE
Update repo to resolve issues and bugs.

### PR DESCRIPTION
**Pull request by nubnubyas**
The following changes were made to resolve the issues mentioned during PE-D:

1. Change finding by appointment to be case insensitive with respect to the month.
2. Update UG to prevent confusion for delete and find command.
3. Update Screenshots for quickstart in UG to match the actual application.
4. Update messages and comments, change from `Person` to `Patient` to avoid confusion.
5. Update find command to handle exception thrown when an empty input is given. e.g. `find n/`

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -48,6 +48,8 @@ HealthSync is a **powerful desktop application designed specifically for clinic
 > 3. Command Box
 > 4. Logger Tab
 
+>:bulb: Some outputs have to be scrolled to view.
```
Comment (kanna-1) [inline docs/UserGuide.md:51]: Should this be rephrased as `Longer outputs have to scrolled to be viewed`.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -570,7 +576,7 @@ JavaScript Object Notation. This is the file format used by HealthSync to save a
 | **List**       | `ls`     | `list`                                                                                                                                         |
 | **Add**        | `a`      | `add n/NAME id/IC_NUMBER [field] ...` <br> e.g., `add n/James Ho id/SXXXX123D p/91234567 a/A Estate, Clementi Rd, 1234665 e/james@example.com` |
 | **Edit**       | `e`      | `edit n/NAME [field]` *or* `edit id/IC_NUMBER [field] ... `<br> e.g.,`edit n/James Lee e/jameslee@example.com`                                 |
-| **Delete**     | `d`      | `delete n/NAME`                                                                                                                                |
+| **Delete**     | `d`      | `delete NAME [field]` *or* `delete id/IC_NUMBER [field] ...` <br> e.g., `delete n/Alex Yeoh m/Diabetes`                                                                                                           |
```
Comment (kanna-1) [inline docs/UserGuide.md:579]: Missing `n/` prefix for NAME.,

File: src/main/java/seedu/address/logic/parser/FindCommandParser.java [hunk 3]
```diff
@@ -47,12 +50,20 @@ public FindCommand parse(String args) throws ParseException {
         CompositePredicate findCommandPredicate = new CompositePredicate();
         if (argMultimap.getValue(PREFIX_NAME).isPresent()) {
             String[] nameKeywords = argMultimap.getValue(PREFIX_NAME).get().trim().split("\\s+");
-            findCommandPredicate.add(new NameContainsKeywordsPredicate(Arrays.asList(nameKeywords)));
+            List<String> nameKeywordsList = Arrays.asList(nameKeywords);
+            if (nameKeywordsList.get(0) == null || nameKeywordsList.get(0).equals("")) {
```
Comment (kanna-1) [inline src/main/java/seedu/address/logic/parser/FindCommandParser.java:54]: Null is never returned from the split method.

File: docs/UserGuide.md (hunk 1 above)
Comment (hyc17003) [reply docs/UserGuide.md:51]: Agreed, think this would be clearer.

**Review by hyc17003 — COMMENTED** [review]
Other than current suggested changes, this LGTM.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -154,9 +156,10 @@ Expected outputs when command fails:
 Deletes the specified patient or an optional fields of the patient from HealthSync.
 
 * Deletes the patient or an optional field of the patient with the specified `n/NAME or id/IC_NUMBER`.
```
Comment (WZWren) [inline docs/UserGuide.md:158]: Can I clarify how we are handling `or`s in our UG? I remember seeing this issue being handled by you during our meeting, will it be in a separate PR?

Comment (nubnubyas) [reply docs/UserGuide.md:158]: It will be resolved in a separate PR.
